app/instance-initializers/head.js

```diff
@@ -1,30 +1,33 @@
 import ENV from '../config/environment';
 
 export function initialize(owner) {
-  if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
+  if (typeof FastBoot === 'undefined') {
+    if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
 
-  // clear fast booted head (if any)
-  let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
-  let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
-  if (startMeta && endMeta) {
-    let el = startMeta.nextSibling
-    while(el && el !== endMeta) {
-      document.head.removeChild(el);
-      el = startMeta.nextSibling;
+    // clear fast booted head (if any)
+    let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
+    let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
+    if (startMeta && endMeta) {
+      let el = startMeta.nextSibling
+      while(el && el !== endMeta) {
+        document.head.removeChild(el);
+        el = startMeta.nextSibling;
+      }
+      document.head.removeChild(startMeta);
+      document.head.removeChild(endMeta);
     }
-    document.head.removeChild(startMeta);
-    document.head.removeChild(endMeta);
-  }
 
-  const component = owner.lookup('component:head-layout');
-  component.appendTo(document.head);
+    const component = owner.lookup('component:head-layout');
+    component.appendTo(document.head);
+  } else {
```

Comment:
From slack conversation, initializers in `/app/(instance-)initializers` will run both in fastboot and the browser, so you have to guard if they do fastboot-incompatible stuff.

Initializers in `/fastboot/(instance-)initializers` will only run in fastboot so there is no need to guard.